utils/tc_marian_logs.py

```diff
@@ -1,16 +1,49 @@
 """
-Downloads Marian training logs for a Taskcluster task group
+Downloads Marian training logs and evaluation results for a Taskcluster task group
 """
 
 import argparse
+import csv
+import enum
 import os
+import re
 
 import taskcluster
-from taskcluster.download import downloadArtifactToFile
+from taskcluster.download import downloadArtifactToBuf, downloadArtifactToFile
+
+TC_MOZILLA = "https://firefox-ci-tc.services.mozilla.com"
+
+# to parse evaluation task tag
+# examples:
+# evaluate-teacher-flores-flores_aug-title_devtest-lt-en-1_2
+# evaluate-teacher-mtdata-mtdata_aug-mix_Neulab-tedtalks_test-1-eng-lit-lt-en-1_2
+# evaluate-finetuned-student-sacrebleu-wmt19-lt-en
+eval_regex = re.compile(
```

Comment:
ideally we should add metadata to Taskcluster tasks, so that we don't need to parse the tag which can be quite different for different tasks

Reply:
Yeah, this is not very maintainable, and will be really brittle.